js/dom_types.ts

```diff
@@ -127,35 +127,40 @@ interface EventListener {
   (evt: Event): void;
 }
 
-interface EventInit {
+export interface EventInit {
   bubbles?: boolean;
   cancelable?: boolean;
   composed?: boolean;
 }
 
-interface Event {
-  readonly bubbles: boolean;
+export interface Event {
+  readonly type: string;
+  readonly target: EventTarget | null;
+  readonly srcElement: EventTarget | null;
+  readonly currentTarget: EventTarget | null;
+  composedPath(): EventTarget[];
+
+  readonly NONE: number;
+  readonly CAPTURING_PHASE: number;
+  readonly AT_TARGET: number;
+  readonly BUBBLING_PHASE: number;
+  readonly eventPhase: number;
+
+  stopPropagation(): void;
   cancelBubble: boolean;
+  stopImmediatePropagation(): void;
+
+  readonly bubbles: boolean;
   readonly cancelable: boolean;
-  readonly composed: boolean;
-  readonly currentTarget: EventTarget | null;
+  returnValue: boolean;
+  preventDefault(): void;
   readonly defaultPrevented: boolean;
-  readonly eventPhase: number;
+  readonly composed: boolean;
+
   readonly isTrusted: boolean;
-  returnValue: boolean;
-  readonly srcElement: Element | null;
-  readonly target: EventTarget | null;
   readonly timeStamp: number;
-  readonly type: string;
-  deepPath(): EventTarget[];
+
   initEvent(type: string, bubbles?: boolean, cancelable?: boolean): void;
```

Comment:
I don't think we should keep this, it's deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent
From WHATWG: *As events have constructors initEvent() is redundant and incapable of setting composed. It has to be supported for legacy content.*

It's also not present in Event's IDL: https://dom.spec.whatwg.org/#interface-event

Reply:
Oh, sorry, I didn't read the PR description well, I'm +1 with @kitsonk's point.